fix(docs): update NuxtHub messaging for multi-platform support (#2260)
nuxt/nuxt.com: content/blog/14.nuxt-on-the-edge.md

**2024 update:** I released [NuxtHub](https://hub.nuxt.com) to let you build full-stack applications with Nuxt on the edge, on your Cloudflare account with zero configuration. It includes a database, blob storage, KV, remote storage and more.
::

::note

> **2026 update:** NuxtHub is now platform-agnostic. It adds database, KV, blob storage, and cache to your Nuxt application and works on Vercel, Cloudflare, Netlify, Deno, and more — not only Cloudflare. Learn more on [hub.nuxt.com](https://hub.nuxt.com).

Reviewer: ```suggestion
**2026 update:** NuxtHub is now multi-providers. It adds database, KV, blob storage, and cache to your Nuxt application and works on Vercel, Cloudflare, Netlify, Deno, and more. Learn more on [hub.nuxt.com](https://hub.nuxt.com).
```
